tarantool/mesh_connection.py

```diff
@@ -69,6 +102,11 @@ def format_error(address, err):
     if 'port' not in address or address['port'] is None:
         return format_error(address, 'port is not set or None')
 
+    # Set default values.
+    for k, v in default_addr_opts.items():
```

Comment:
So now you change input argument? It doesn't seem like a good pattern

Reply:
Yes, you are right. Not it does not.